src/domains/afm-app/api/calculator-service/node/v1/_base_policy.xml

```diff
@@ -0,0 +1,106 @@
+    - Policies are applied in the order of their appearance, from the top down.
+    - Comments within policy elements are not supported and may disappear. Place your comments between policy elements or at a higher level scope.
+-->
+<policies>
+  <inbound>
+    <base />
+    <set-backend-service base-url="https://${hostname}/pagopa-afm-calculator-service" />
+
+    <set-variable name="isMock" value="@(context.Request.Headers.GetValueOrDefault("Ocp-Apim-Subscription-Key","").Equals("{{afm-dev-secondary-sub-key}}") || true)" />
+    <choose>
+      <when condition="@(context.Variables.GetValueOrDefault<bool>("isMock"))">
+      <!--Mock Responses-->
+      <set-variable name="statusCode" value="@(context.Request.Headers.GetValueOrDefault("Ocp-Apim-Error-code", "200"))" />
+      <set-variable name="scenario1" value="@(context.Request.Body.As<JObject>(true)["touchpoint"].Value<string>().Equals("ATM"))" />
+      <set-variable name="scenario2" value="@(context.Request.Body.As<JObject>(true)["touchpoint"].Value<string>().Equals("ANY"))" />
+      <set-variable name="scenario3" value="@(context.Request.Body.As<JObject>(true)["touchpoint"].Value<string>().Equals("BETTING"))" />
+      <!--Scenario 1-->
+      <choose>
+        <when condition="@(context.Variables.GetValueOrDefault<bool>("scenario1"))">
+        <return-response>
+          <set-status code="@(int.Parse((string)context.Variables["statusCode"]))" />
+          <set-header name="Content-Type" exists-action="override">
+            <value>application/json</value>
+          </set-header>
+          <set-body>[
+            {
+            "taxPayerFee": 130,
+            "primaryCiIncurredFee": 20,
+            "paymentMethod": "PO",
+            "touchpoint": "ATM",
+            "idBundle": "1",
+            "idCiBundle": "1",
+            "idPsp": "88888888888"
+            }
+            ]</set-body>
+        </return-response>
+      </when>
+    </choose>
+    <!--Scenario 2-->
+    <choose>
+      <when condition="@(context.Variables.GetValueOrDefault<bool>("scenario2"))">
+      <return-response>
+        <set-status code="@(int.Parse((string)context.Variables["statusCode"]))" />
+        <set-header name="Content-Type" exists-action="override">
+          <value>application/json</value>
+        </set-header>
+        <set-body>[
+          {
+          "taxPayerFee": 80,
+          "primaryCiIncurredFee": 0,
+          "paymentMethod": "PO",
+          "touchpoint": null,
+          "idBundle": "2",
+          "idCiBundle": null,
+          "idPsp": "88888888888"
+          },
+          {
+          "taxPayerFee": 130,
+          "primaryCiIncurredFee": 20,
+          "paymentMethod": "PO",
+          "touchpoint": "ATM",
+          "idBundle": "1",
+          "idCiBundle": "1",
+          "idPsp": "88888888888"
+          }
+          ]</set-body>
+      </return-response>
+    </when>
+    </choose>
+    <!--Scenario 3-->
+    <choose>
```

Comment:
This is _scenario 4_
_Scenario 3_ is missing

Reply:
[scenario 3](https://pagopa.atlassian.net/wiki/spaces/PPR/pages/550634053/Mock#response.2) has empty list as response